SwordImportExportPlugin.inc.php

```diff
@@ -173,9 +196,27 @@ public function display($args, $request) {
 							if ($depositGalleys) $deposit->addGalleys();
 							if ($depositEditorial) $deposit->addEditorial();
 							$deposit->createPackage();
-							$response = $deposit->deposit($swordDepositPoint, $username, $password);
+							$response = $deposit->deposit(
+								$swordDepositPoint,
+								$username,
+								$password,
+								$request->getUserVar('swordApiKey'));
 							switch ($response->sac_status) {
 							case 200:
+								$stmt_link = array_shift(
+									array_filter($response->sac_links, function($link) {
+										return $link->sac_linkrel == 'http://purl.org/net/sword/terms/statement';
+									}));
+								$stmt_href = $stmt_link->sac_linkhref->__toString();
+								$data = $publishedArticle->getAllData();
+								$ssi = [];
+								if (array_has($data, 'swordStatementIri')) {
+									$ssi = json_decode($data['swordStatementIri'], true);
+								}
+								$ssi[$depositPointId] = $stmt_href;
+								$publishedArticle->setData('swordStatementIri', json_encode($ssi));
```

Comment:
There is a data object setting type of `Object`, which uses PHP's serialization instead of JSON. Just for consistency's sake, maybe use that instead of a `setting_type` of `string` and a value that's JSON-encoded.

Reply:
Done!